core/src/navigation/ContextSwitcherNav.html

```diff
@@ -105,15 +105,18 @@
     }
   }
 
-  .fd-menu__list--top {
-    border-bottom: 1px solid #eeeeef;
-    border-bottom-left-radius: 0;
-    border-bottom-right-radius: 0;
-  }
-  .fd-menu__list--bottom {
-    border-top: 1px solid #eeeeef;
-    border-top-left-radius: 0;
-    border-top-right-radius: 0;
+  .fd-menu__list {
+    box-shadow: none;
```

Comment:
If I understand correctly, you could remove this custom style and apply the fundamental class `fd-menu__list--no-shadow`